ocaml/xapi/quicktest_cbt.ml

```diff
@@ -130,6 +164,8 @@ let test ~session_id =
       [
         (fun ~vDI -> vdi_data_destroy_test ~session_id ~vDI) ,
         [ `vdi_enable_cbt ; `vdi_disable_cbt ; `vdi_data_destroy ; `vdi_snapshot ]
+      ; (fun ~vDI -> vdi_clone_copy_test ~session_id ~sR ~vDI),
+        [ `vdi_enable_cbt ; `vdi_create ; `vdi_clone ; `vdi_snapshot ]
```

Comment:
Same as above:  we're not calling VDI.snapshot, so the SR doesn't need to support it.